src/common/catalog-entities/web-link.ts

```diff
@@ -34,12 +34,13 @@ export class WebLink extends CatalogEntity<CatalogEntityMetadata, WebLinkStatus,
     // NOTE: this is safe because `onContextMenuOpen` is only supposed to be called in the renderer
     const di = getEnvironmentSpecificLegacyGlobalDiForExtensionApi(Environments.renderer);
```

Comment:
Side-note: `getEnvironmentSpecificLegacyGlobalDiForExtensionApi` -> the part about "ForExtensionApi" hasn't been very true for some time, and we should remove it ;)

Reply:
Sure, future